apps/app-frontend/src/store/featuredProjects.js

```diff
@@ -0,0 +1,49 @@
+import { defineStore } from 'pinia'
+import { get_search_results } from '@/helpers/cache.js'
+
+export const useFeaturedProjects = defineStore('featuredProjects', {
+  state: () => ({
+    modpacks: undefined,
+    mods: undefined,
+  }),
+  actions: {
+    async getFeaturedModpack(filter) {
+      if (this.modpacks !== undefined && this.modpacks.length !== 0) {
```

Comment:
Couldn't this be simplified to `if (this.modpacks?.length) {`?
See [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_operators#optional_chaining)

```suggestion
      if (this.modpacks?.length) {
```